recipes/cista/all/patches/0.11-0001-use-and-operator.patch

```diff
@@ -0,0 +1,32 @@
+diff --git a/cista.h b/cista.h
```

Comment:
I don't see this patch listed in conandata.yml 🤔

Reply:
@uilianries 
I missed to delete it.
Thanks a lot.